src/core_init_atmosphere/Registry.xml

```diff
@@ -191,9 +196,14 @@
                      description="The supersampling factor to be used for MODIS maximum snow albedo and monthly albedo datasets (case 7 only)"
                      possible_values="Positive integer values"/>
 
+	        <nml_option name="config_supersample_fac_lu"    type="integer"       default_value="1"
```

Comment:
Just as an idea, what if we called this `config_lu_supersample_factor`? That name would match the pattern set by the two existing super-sampling options nicely: `config_supersample_factor`, `config_lu_supersample_factor`, and `config_30s_supersample_factor`.

Reply:
Thank you for the idea. The name is changed as you suggested.